CHANGELOG.md

```diff
@@ -1,26 +1,71 @@
-## 1.5.5 (May 1, 2020)
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [1.5.7](https://github.com/simplepie/simplepie/compare/1.5.6...1.5.7) - 2021-12-19
+
+* Fix PHP8 crash due to insufficient isset test by @Alkarex in #670
+* gitignore tests by @Alkarex in #671
+* Reduce memory when parsing large feeds by @Alkarex in #672
```

Comment:
It would be useful to use issue links like in the previous releases since GitHub does not linkify shortlinks nor would they be clickable when viewing the changelog locally.

Reply:
You are right, older entries are also linked to the issues. Thank you. I will fix this. 👍

Reply:
I've added links to the issues.